src/load/azext_load/data_plane/utils/utils.py

```diff
@@ -284,8 +284,8 @@ def convert_yaml_to_test(data):
     if "description" in data:
         new_body["description"] = data["description"]
     new_body["keyvaultReferenceIdentityType"] = IdentityType.SystemAssigned
-    if "keyvaultReferenceIdentityId" in data:
-        new_body["keyvaultReferenceIdentityId"] = data["keyvaultReferenceIdentityId"]
+    if "keyVaultReferenceIdentity" in data:
```

Comment:
For my understanding do these keys need to be case sensitive or do they need to be case-insensitive ?

Reply:
looks like they are case sensitive as of now, since we don't have explicit conversions to lowercase. In this case the key was different to what is mentioned in the docs for config file. We were looking for `keyvaultReferenceIdentityId` while the key is `keyVaultReferenceIdentity` for config file